js/src/forum/components/PostStream.js

```diff
@@ -42,6 +42,7 @@ export default class PostStream extends Component {
         content = PostComponent ? PostComponent.component({ post }) : '';
 
         attrs.key = 'post' + post.id();
+        attrs.oncreate = (vnode) => setTimeout(() => $(vnode.dom).removeClass('fadeIn'), 500); // We give enough time for the animation to complete
```

Comment:
Mithril-wise wouldn't it be better to insert the `fadeIn` class via jQuery here? Right now I think if an extension triggers a full redraw of the page, the `fadeIn` class would be re-added.

Something like

```
attrs.oncreate = (vnode) => {
    $(vnode.dom).addClass('fadeIn');
    setTimeout(() => $(vnode.dom).removeClass('fadeIn'), 500)
};
```

I'd suggest a comment that references the original duration being CSS. Something like "// Duration of the fadeIn CSS animation + 100ms so it has time to complete"